go.mod

```diff
@@ -3,12 +3,14 @@ module github.com/zcashfoundation/dnsseeder
 go 1.14
 
 require (
-	github.com/btcsuite/btcd v0.20.1-beta
+	github.com/btcsuite/btcd v0.22.0-beta
 	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
 	github.com/caddyserver/caddy v1.0.5
 	github.com/coredns/coredns v1.6.9
 	github.com/miekg/dns v1.1.29
 	github.com/pkg/errors v0.9.1
 )
 
-replace github.com/btcsuite/btcd => github.com/gtank/btcd v0.0.0-20191012142736-b43c61a68604
+// Currently pointing to "addrv2" branch
+// TODO: change to "main-zfnd" branch after https://github.com/ZcashFoundation/btcd/pull/1 is merged
+replace github.com/btcsuite/btcd => github.com/ZcashFoundation/btcd v0.22.0-beta.0.20211116150640-079ebf598ccb
```

Comment:
When we merge https://github.com/ZcashFoundation/btcd/pull/1 , GitHub might delete the branch with this commit, and clean up the commit.

So we either need to:
* update to `main-zfnd` before we merge this PR, or
* tag the commit we're depending on.

Reply:
Agreed, I think it's easier to update to `main-zfnd` just after https://github.com/ZcashFoundation/btcd/pull/1 is accepted.